packages/python-packages/low-level-client-api-gen/llc_api/llc_view_models.py

```diff
@@ -0,0 +1,547 @@
+        self.operations=operations; #parameterview list
+        self.Tokens =[]
+        self.indent = 0 
+        self.namespace = namespace
+    
+    @classmethod
+    def from_yaml(cls,yaml_data: Dict[str,Any],op_group,name): 
+            o = []
+            for i in range(0,len(yaml_data["operationGroups"][op_group]["operations"])):
+                o.append(LLCOperationView.from_yaml(yaml_data,op_group,i,name))
+            return cls(
+                operation_group_name = yaml_data["operationGroups"][op_group]["language"]["default"]["name"],
+                operations = o,
+                namespace=name,
+            )
+
+    def get_tokens(self):
+            return self.Tokens
+
+    def add_token(self, token):
+        self.Tokens.append(token)
+
+    #have a to_token to create the line for parameters
+    def to_token(self):
+
+        #Each operation will indent itself by 4
+        self.add_new_line()
+        
+
+        if self.operation_group:
+            self.add_whitespace(1)
+            #Operation Name token
+            self.add_keyword(self.namespace+self.operation_group,self.operation_group,self.namespace+self.operation_group)
+
+            self.add_new_line()
+    
+
+            for operation in range(0,len(self.operations)):
+                if self.operations[operation]:
+                    self.operations[operation].to_token()
+                    if operation==0:
+                        self.add_whitespace(2)
+                        self.add_punctuation("{")
+                    self.add_new_line()
+                    self.add_whitespace(2)
+                    for t in self.operations[operation].get_tokens():
+                        self.add_token(t)
+            self.add_whitespace(2)
+            self.add_punctuation("}")
+            self.add_new_line(1)
+        else:
+            for operation in range(0,len(self.operations)):
+                if self.operations[operation]:
+                    self.operations[operation].to_token()
+                    for t in self.operations[operation].get_tokens():
+                        self.add_token(t)
+
+       
+
+    def to_json(self):
+        obj_dict={}
+        self.to_token()
+        for key in OP_FIELDS:
+            obj_dict[key] = self.__dict__[key]
+        return obj_dict
+
+
+class LLCOperationView(FormattingClass):
+    def __init__(self, operation_name, parameters,namespace, description ="", paging = "",lro=""):
+        self.operation=operation_name;
+        self.parameters=parameters; #parameterview list
+        self.Tokens =[]
+        self.indent = 0 
+        self.namespace = namespace
+        self.description = description
+        self.paging = paging
+        self.lro = lro
+
+    @classmethod
+    def from_yaml(cls,yaml_data: Dict[str,Any],op_group,num,name): 
+            param = []
+            if len(yaml_data["operationGroups"][op_group]["operations"][num]["signatureParameters"])==0:
```

Comment:
in python you can simply say

```suggestion
            if yaml_data["operationGroups"][op_group]["operations"][num]["signatureParameters"]:
```

and if the list is empty it will eval to False